### ISSUE TITLE
Rundeck fails to start on OpenJDK 8_201

### ISSUE THREAD

**Issue by Ziemowit-LH** [rundeck/rundeck#6241]
**Describe the bug**
Installation/first launch of Rundeck fails on OpenJDK 1.8.0_201

**My Rundeck detail**
* Rundeck version: 3.3.0-20200701
* install type: war
* OS Name/version: Windows 10 1909
* DB Type/version: n/a, builtin

**To Reproduce**
Download the war,
Place in empty directory, launch with:
```
   set RDECK_BASE=%~dp0
   java -Xmx1024m -jar rundeck-3.3.0-20200701.war

```
**Expected behavior**
Should extract and start new instance. https://docs.rundeck.com/docs/administration/install/system-requirements.html states OpenJDK 8 is supported.

**Screenshots**
Logfile
[firstlaunch.zip](https://github.com/rundeck/rundeck/files/4896508/firstlaunch.zip)

**Desktop (please complete the following information):**
n/a

**Additional context**
java -version output:
openjdk version "1.8.0_201-2-redhat"
OpenJDK Runtime Environment (build 1.8.0_201-2-redhat-b09)
OpenJDK 64-Bit Server VM (build 25.201-b09, mixed mode)

**Comment by sjrd218** [comment]
This issue will be fixed with PR #6215.

**Comment by sjrd218** [comment]
Duplicate of #6214.
